components/instructions/instructionPanel.tsx

```diff
@@ -43,6 +86,29 @@ export function InstructionPanel() {
                       index={idx + 1}
                       proposalInstruction={pi.info}
                     ></InstructionCard>
+
+                    {pi.info.executionStatus ===
+                      InstructionExecutionStatus.Success &&
+                      notify({
+                        type: 'success',
+                        message: 'Instruction executed successful',
+                        description: 'Instruction executed successful',
+                      })}
+
+                    {showExecuteInstructionModal && (
+                      <Execute
+                        isOpen={showExecuteInstructionModal}
+                        onClose={() => setShowExecuteInstructionModal(false)}
+                        instruction={pi}
+                      />
+                    )}
+
+                    <p
```

Comment:
Let's make it to look like a button and not link 

<img width="265" alt="image" src="https://user-images.githubusercontent.com/5817016/146169059-83e4d63b-c56d-40d0-a85e-264e321d344b.png">